cfgov/ask_cfpb/tests/test_search.py

```diff
@@ -132,7 +132,8 @@ def test_ask_search_en_suggestion(self, mock_search):
 
     @mock.patch.object(AnswerPageDocument, "search")
     def test_ask_search_autocomplete_honors_max_chars(self, mock_search):
-        valid_term = "You saw the masterpiece, she looks a lot like you!"
+        valid_term = "This search has 75 characters, the upper limit we have "
+        "set for Ask searches"
```

Comment:
```suggestion
        valid_term = "Years, days, makes no difference to me babe "
        "You look exactly the same to me"
```

Reply:
Can't believe I almost ruined the theme. Fixed in 9d1588.